src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,32 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = {"/index", "/"})
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init() {
+        myResource = MyCoolResource.openResource();
+    }
+
+    @Override
+    public void destroy() {
+        myResource.close();
+    }
+
+    @Override
```

Comment:
let's stick to the order: init -> service -> destroy

Reply:
Changes completed